### ISSUE TITLE
JSON-LD: Extra, unresolvable classes are added

### ISSUE THREAD

**Issue by azaroth42** [archesproject/arches#3853]
There are additional classes added to the data by the platform that don't exist.

For example, I create an object and get:

```
"type": ["ManMadeObject", "http://localhost:8000/graph/7718ca0c-9b37-11e8-895f-14b1c8006236"]
```

Trying to request that URI returns a 500 error, as GraphManagerView returns None, not an HttpResponse.

I would expect:

`"type": "ManMadeObject"`

**Comment by apeters** [comment]
When POSTing a new resource I use the graph id to know what resource instance to create.
ManMadeObject wouldn't be sufficient by itself to reliably determine the resource type.

It looks like we need to make an endpoint for that URL (http://localhost:8000/graph/7718ca0c-9b37-11e8-895f-14b1c8006236).  I could have sworn we had one.  :|

**Comment by azaroth42** [comment]
:+1: to having an endpoint per model to make it easy to determine which model to create.

**Comment by apeters** [comment]
For now the fix for this is to add an endpoint for graphs so that at least the URL resolves to something.
Do we need JSON-LD as an output format for this as well?

**Comment by azaroth42** [comment]
I'm not sure what the question is? The JSON-LD output should only have one `type`, the class of the node in the graph. The endpoint is a protocol consideration, and should not be in the representation.

**Comment by apeters** [comment]
I need the graph id to identify the graph.  I could have several graphs that are of type `ManMadeObject`.

**Comment by apeters** [comment]
Currently there is only 1 endpoint for ALL resources types

```
http://localhost/resources
or 
http://localhost/resources/{resource instance id}
```

To do what you're suggesting would require that we create a subpath in our endpoint that can resolve to a specific graph type.

```
http://localhost/resources/{graph id}/{optional resource instance id}
or 
http://localhost/resources/{graph name}/{optional resource instance id} <-- but only if we enforced name uniqueness, which we don't
```

**Comment by apeters** [comment]
@azaroth42 is this what you had in mind?

**Comment by azaroth42** [comment]
Yes, exactly :) Or, even better, have the graph configuration allow a more friendly (but still unique) slug to be used instead of {graph id}. The name of the graph is likely to contain spaces, which would not be good for the URI of the endpoint.
